.github/workflows/ci.yaml

```diff
@@ -236,17 +234,24 @@ jobs:
                   ENGFLOW_CLIENT_CRT: ${{ secrets.ENGFLOW_CLIENT_CRT }}
                   ENGFLOW_PRIVATE_KEY: ${{ secrets.ENGFLOW_PRIVATE_KEY }}
 
-            - name: Set bzlmod flag
+            - name: Set bzlmod flag (for integration tests)
               # Store the --enable_bzlmod flag that we add to the test command below
               # only when we're running bzlmod in our test matrix.
               id: set_bzlmod_flag
-              if: matrix.bzlmodEnabled
+              if: ${{ matrix.bzlmodEnabled && matrix.folder != '.'}}
               run: echo "bzlmod_flag=--enable_bzlmod" >> $GITHUB_OUTPUT
 
+            - name: Set bzlmod configuration (for test //...)
+              # Store the --enable_bzlmod flag that we add to the test command below
+              # only when we're running bzlmod in our test matrix.
+              id: set_bzlmod_config
+              if: ${{ matrix.bzlmodEnabled && matrix.folder == '.'}}
+              run: echo "bzlmod_config=--config=bzlmod" >> $GITHUB_OUTPUT
+
```

Comment:
In the case of `test //...` we need to set `--config=bzlmod` rather than just an `--enable_bzlmod` flag.